setup.py

```diff
@@ -79,6 +79,7 @@ def run(self):
     "shortuuid>=0.5.0",
     "tqdm>=4.40.0,<5",
     "packaging>=19.0",
+    "zc.lockfile>=1.2.1",
```

Comment:
Note to self: no need to update this in our conda package, as I've left zc.lockfile there in preparation for this PR.